ledger/src/bigtable_upload.rs

```diff
@@ -15,18 +15,32 @@ use {
     },
 };
 
-// Attempt to upload this many blocks in parallel
-const NUM_BLOCKS_TO_UPLOAD_IN_PARALLEL: usize = 32;
+pub struct ConfirmedBlockUploadConfig {
+    pub force_reupload: bool,
+    // number of blocks uploaded in parallel
```

Comment:
This comment seems redundant with the field name. Is there anything distinct we want to say here?